mscore/file.cpp

```diff
@@ -2144,13 +2143,8 @@ bool MuseScore::savePdf(QList<Score*> cs_, const QString& saveName)
       p.setRenderHint(QPainter::Antialiasing, true);
       p.setRenderHint(QPainter::TextAntialiasing, true);
 
-      p.setViewport(QRect(0.0, 0.0, size.width() * pdfWriter.logicalDpiX(),
-         size.height() * pdfWriter.logicalDpiY()));
-      p.setWindow(QRect(0.0, 0.0, size.width() * DPI, size.height() * DPI));
-
       double pr = MScore::pixelRatio;
-      MScore::pixelRatio = DPI / pdfWriter.logicalDpiX();
-      MScore::pdfPrinting = true;
+      //MScore::pdfPrinting = true;
```

Comment:
You left some commented code.
If it is not needed anymore, you could remove it (we have history in git).
If it is needed, it should probably be uncommented.

Reply:
It's a good observation.  In this case, I left the commented code because in my experience, some of these sort of things turn out to be system-dependent, and I thought it important to make it easier for others to try their own variations if it turns out my fix breaks something for them.  That's what I meant when I said I did this "for safety (?)" - I'm not 100% sure it really is the right thing to do.  This is the sort of thing we do typically use commenting out (as opposed to removing entirely) for.  But it's also no problem to remove them after review if that is the consensus.

Reply:
It is better to revise the usage of the `MScore::pdfPrinting`. The only place where it is used is `libmscore/sym.cpp:5650` to define proper font and its size.

If you remove assigning `true` to the variable, it will affect drawing all symbols.

Reply:
Right, but drawing doesn't happen during layout, only during the call to print() later.  And print *already* sets pdfPrinting.  It didn't originally when this code was written, so that's probably why these calls were there, but they aren't needed any more.  In fact I think they can be removed from other functions that call Socre::print().

Iffor some reason we *do* recide it is import to have pdfPrinting set to true during lsyout, then we actually need to move this assignment inside the loop, but Score:print() sets it back to false when it is done.  So it was only actually set to true for the first score laid out, then it was false while laying out the others.